eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -57,6 +57,7 @@ case class NodeParams(keyManager: KeyManager,
                       maxAcceptedHtlcs: Int,
                       expiryDeltaBlocks: CltvExpiryDelta,
                       fulfillSafetyBeforeTimeoutBlocks: CltvExpiryDelta,
+                      minFinalExpiryDeltaBlocks: CltvExpiryDelta,
```

Comment:
The field `minFinalExpiryDelta` is typed, no need to specify that the unit is in blocks.

```suggestion
                      minFinalExpiryDelta: CltvExpiryDelta,
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1483/commits/135f40e29b8c1ef0cc7fe2ebfd4f13fdb45cc472